querydsl-sql/src/main/java/com/querydsl/sql/types/AbstractJodaTimeDateTimeType.java

```diff
@@ -0,0 +1,35 @@
+package com.querydsl.sql.types;
+
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+
+import java.util.Calendar;
+import java.util.TimeZone;
+
+/**
+ * Common abstract superclass for Type implementations for Joda-Time
+ *
+ * @param <T>
+ */
+public abstract class AbstractJodaTimeDateTimeType<T> extends AbstractType<T> {
```

Comment:
Why not:

```suggestion
public abstract class AbstractJodaTimeDateTimeType<T> extends AbstractDateTimeType<T> {
```

here?